dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/resolver/FeatureResolver.java

```diff
@@ -143,8 +143,21 @@ public static class Chain {
          * @param originalFeatureReq The full feature name that is required.
          */
         public Chain(Collection<String> chain, List<String> candidates, String preferredVersion, String originalFeatureReq) {
-            _chain = chain.isEmpty() ? Collections.<String> emptyList() : Collections.unmodifiableList(new ArrayList<String>(chain));
-            _candidates = Collections.unmodifiableList(candidates);
```

Comment:
Why don't we want the Collection to be unmodifiable?  Collections.singletonList is unmodifiable in the new method.